addons/website/models/mixins.py

```diff
@@ -112,6 +112,23 @@ def _default_cover_properties(self):
             "resize_class": "o_half_screen_height",
         }
 
+    def write(self, vals):
+        result = True
+        for item in self:
+            copy_vals = dict(vals)
+            # Prevent updates on list displays from destroying resize_class.
+            if 'cover_properties' in copy_vals:
+                cover_properties = json_scriptsafe.loads(copy_vals['cover_properties'])
+                resize_class = cover_properties.get('resize_class', '').split()
+                if 'o_half_screen_height' not in resize_class and \
+                        'o_full_screen_height' not in resize_class and \
+                        'cover_auto' not in resize_class:
+                    old_cover_properties = json_scriptsafe.loads(item.cover_properties)
+                    cover_properties['resize_class'] = old_cover_properties.get('resize_class', 'o_half_screen_height')
+                    copy_vals['cover_properties'] = json_scriptsafe.dumps(cover_properties)
+            result &= super(WebsiteCoverPropertiesMixin, item).write(copy_vals)
+        return result
```

Comment:
@bso-odoo I am not sure it is good to change the multi-write into a write on each record one by one in all case, especially in a mixin model, maybe the ORM is somehow smart and make something out of it but would not risk it anyway :wink: I'll change that.